cognite/neat/core/data_classes/transformation_rules.py

```diff
@@ -424,6 +424,24 @@ def get_classes_with_properties(self) -> dict[str, Property]:
 
         return class_property_pairs
 
+    def get_class_property_pairs(self) -> dict[str, dict[str, Property]]:
+        """This method will actually consider only the first definition of given property!"""
+        class_property_pairs = {}
+
+        for class_, properties in self.get_classes_with_properties().items():
```

Comment:
Note the method `get_classes_with_properties` has wrong return type, it is `dict[str, Property]` but should be `dict[str, list[Property]`. That is causing the linter to complain in this method.

Reply:
fixed